.eslintrc

```diff
@@ -32,6 +32,8 @@
         "@typescript-eslint/no-empty-function": "off",
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-namespace": "off",
+        "no-async-promise-executor": "off",
```

Comment:
I'm fine with disabling the rule in general but another option would be to just override it for these two functions

Reply:
there are multiple places where async/await is preferred and required (even in Role verifier). disabling looks like a better option.